File: app/src/main/java/jgappsandgames/smartreminderslite/tasks/ChildrenAdapter.java
package jgappsandgames.smartreminderslite.tasks;

// Java
import java.util.ArrayList;

// App
import jgappsandgames.smartreminderslite.adapter.TaskAdapterInterface;

/**
 * ChildrenAdapter
 * Created by joshua on 8/31/17.
 */
class ChildrenAdapter extends TaskAdapterInterface {
    // Initializer
    ChildrenAdapter(TaskActivity activity, ArrayList<String> tasks) {
        super(activity, activity, tasks, null);
    }
}

File: app/src/main/java/jgappsandgames/smartreminderslite/tasks/ChildrenAdapter.kt
package jgappsandgames.smartreminderslite.tasks

// App
import jgappsandgames.smartreminderslite.adapter.TaskAdapterInterface
import jgappsandgames.smartreminderssave.tasks.TaskManager

/**
 * ChildrenAdapter
 * Created by joshua on 12/16/2017.
 *
 * Displays the Tasks in the Folder View
 */
class ChildrenAdapter(activity: TaskActivity, tasks: ArrayList<String>): TaskAdapterInterface(activity, activity, TaskManager.getTasks(tasks))